CMakeLists.txt

```diff
@@ -42,5 +42,8 @@ install(DIRECTORY launch
         PATTERN "*~" EXCLUDE)
 
 # for Python scripts
-catkin_install_python(PROGRAMS scripts/depth_camera_sonar.py
+catkin_install_python(PROGRAMS
+  scripts/depth_camera_sonar.py
+  src/simple_box_motion.py
+  src/simple_motion.py
```

Comment:
Considering section 1.3 http://wiki.ros.org/UsingPython3/SourceCodeChanges, why not also add `sonar_ros.py`? I understand not including `sonar_equations.py` as it is only imported, but `depth_camera_sonar_basic.launch` calls `sonar_ros.py`.